src/Commands/AdminCommands/CleanupCommand.php

```diff
@@ -327,7 +351,7 @@ private function getQueries($settings)
     public function execute()
     {
         $message = $this->getMessage();
-        $chat_id = $message->getFrom()->getId();
+        $chat_id = $message->getChat()->getId();
```

Comment:
Yes, bot can delete user messages when he has the permissions to do so.

That would be a nice solution!